packages/composer-editor/components/Editor/index.jsx

```diff
@@ -7,13 +7,7 @@ import styled from 'styled-components';
 import { grayDarker } from '@bufferapp/ui/style/colors';
 import { fontSize, fontFamily } from '@bufferapp/ui/style/fonts';
 
-const Editor = styled(Editable)`
-  color: ${grayDarker};
-  padding: 8px;
-  line-height: 24px;
-  font-size: ${fontSize};
-  font-family: ${fontFamily};
-`;
+import EmojiPicker from '../Plugins/EmojiPicker';
```

Comment:
```suggestion
import EmojiPicker from '../plugins/EmojiPicker';
```

Can we use lowercase here for directories unless it's a component folder? I think that would be a little more consistent.

Reply:
Yes, you are right! It took me a couple of commits because I was in a non-case-sensitive system 😅